tests/test_prtype.py

```diff
@@ -0,0 +1,216 @@
+from bedevere import prtype
+
+
+class FakeGH:
+
+    def __init__(self, *, getiter=None, getitem=None, post=None):
+        self._getitem_return = getitem
+        self._getiter_return = getiter
+        self._post_return = post
+        self.getitem_url = None
+        self.getiter_url = None
+        self.post_url = []
+        self.post_data = []
+
+    async def getitem(self, url):
+        self.getitem_url = url
+        return self._getitem_return
+
+    async def post(self, url, *, data):
+        self.post_url.append(url)
+        self.post_data.append(data)
+        return self._post_return
+
+
+GOOD_BASENAME = '2017-06-16-20-32-50.bpo-1234.nonce.rst'
+
+
+
+async def no_files():
```

Comment:
Just need to prefix this with `test_`, then that should take care of coverage.

Reply:
Wow, thank you so much!  That fixed it.  Apologies for not catching it myself.